spec/lib/extensions/ar_region_spec.rb

```diff
@@ -125,4 +125,71 @@
       expect(ManageIQ::Providers::Vmware::InfraManager::Vm.new.split_id).to eq([ManageIQ::Providers::Vmware::InfraManager::Vm.my_region_number, nil])
     end
   end
+
+  describe ".my_region_number" do
+    before do
+      MiqDatabase.delete_all
+    end
```

Comment:
My comment was lost here, but this should not be needed.  If it is, then there are other problems, because the tables should be empty at this point.